schema.graphql

```diff
@@ -485,11 +544,78 @@ type MatchQuestionBuzz {
 """
 Docs group: Match
 """
-type MatchQuestionBonus {
+type BuzzPosition {
+  """
+  The time of the buzz. An ISO 8601-formatted string, with time zone, in the tournament's local time zone. May not be before the start time of reading the relevant question.
+  """
+  time: Date
+
+  """
+  The (`0`-based) index of the word on which the player buzzed.
   """
-  Which bonus number this was. May or may not match the `question_number` of the `MatchQuestion` parent, depending on whether bonuses are paired with tossups or simply read in order (and if the latter, on whether any bonus-earning questions failed to lead to a bonus).
+  word_index: Int
+
+  """
+  The (`0`-based) index of the clue on which the player buzzed. Clues may be defined in the `Question` object.
+  """
+  clue_index: Int
+}
+
+"""
+This object can be reused for every time the same question is read, i.e., it represents a question in general rather than a question being read in a specific match.
+
+Docs group: Match
```

Comment:
Methinks it's best to put `Question` in its own docs group (along with `Packet`?) and maybe move `Player` and `Team` to the `Registration` group.

Reply:
Just `Question`, or also some other children?

Speaking of organization, is there a way to split up `schema.graphql` into multiple files?

Reply:
I was thinking Question and everything that neatly fits with it. This pull request neither defines nor needs a `Packet` type, but such a type in a future version could go into that file as well.

(Do you want to tackle that one now, since it would be best done with a backwards-incompatible change to the definition of the `packets` field on `Round`?)

There's no official way to import one `.graphql` file into another. There's a [proposal that mostly just puts the syntax from JavaScript into a comment](https://oss.prisma.io/content/graphql-import/overview) and we could certainly use that here.

Reply:
Resolving this conversation since the part about splitting up the file is covered by #86.